.packit.yaml

```diff
@@ -12,6 +12,8 @@ packages:
   container-selinux-centos:
     pkg_tool: centpkg
     specfile_path: rpm/container-selinux.spec
+  container-selinux-rhel:
+    specfile_path: rpm/container-selinux.spec
```

Comment:
`files_to_sync` change is coming later I guess?

Reply:
yes